laceworksdk/api/cloud_accounts.py

```diff
@@ -0,0 +1,210 @@
+# -*- coding: utf-8 -*-
+"""
+Lacework Cloud Accounts API wrapper.
+"""
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class CloudAccountsAPI(object):
+    """
+    Lacework Cloud Accounts API.
+    """
+
+    def __init__(self, session):
+        """
+        Initializes the CloudAccountsAPI object.
+
+        :param session: An instance of the HttpSession class
+
+        :return CloudAccountsAPI object.
+        """
+
+        super(CloudAccountsAPI, self).__init__()
+
+        self._session = session
+
+    def create(self,
+               name,
+               type,
+               enabled,
+               data,
+               org=False):
+        """
+        A method to create a new cloud account.
+
+        :param name: A string representing the cloud account name.
+        :param type: A string representing the cloud account type.
+        :param enabled: An integer representing whether the cloud account is enabled.
```

Comment:
nit.  would it be better to use a python boolean here and just translate for `data`  if it needs an int?